curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa371beacea3f48c6300af.md

```diff
@@ -33,6 +33,12 @@ assert(
 );
 ```
 
+Your `h3` element should have a closing tag. Closing tags have a `/` just after the `<` character.
+
+```js
+assert(code.match(/<\/h3>/g).length === 1);
```

Comment:
can you use a specific assert method here? 
https://www.chaijs.com/api/assert/

maybe `lengthOf`?